command/cp.go

```diff
@@ -379,9 +397,11 @@ func (c Copy) doUpload(ctx context.Context, srcurl *url.URL, dsturl *url.URL) er
 
 	dstClient := storage.NewClient(dsturl)
 
-	metadata := map[string]string{
-		"StorageClass": string(c.storageClass),
-		"ContentType":  guessContentType(f),
+	metadata := storage.Metadata{
+		"StorageClass":     string(c.storageClass),
```

Comment:
let's start chain methods to avoid handwriting these keys now. E.g.:

```go
storage.NewMetadata().
    SetStorageClass(...).
    SetContentType(...).
    SetEncryptionMethod(...).
    SetEncryptionKeyID(...)